template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Extensions/IntegratedWebClientOptionsSetup.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Identity.Service.IntegratedWebClient.Extensions
+{
+    public class IntegratedWebClientOptionsSetup : IConfigureOptions<IntegratedWebClientOptions>
```

Comment:
You can slightly simply all of the config binding setups if you derive from ConfigureOptions instead, see: https://github.com/aspnet/Identity/blob/dev/src/Microsoft.AspNetCore.Identity/IdentityServiceCollectionExtensions.cs#L20